pkg/loki/modules.go

```diff
@@ -1242,6 +1244,15 @@ func (t *Loki) initIndexGatewayRing() (_ services.Service, err error) {
 	return t.indexGatewayRingManager, nil
 }
 
+func (t *Loki) initIndexGatewayInterceptors() (services.Service, error) {
+	// Only expose per-tenant metric if index gateway runs as standalone service
+	if util.StringsContain(t.Cfg.Target, IndexGateway) {
```

Comment:
We actually use this to check if the module was set in the targets. Although it does the same thing under the hood, it would be good to keep things consistent.
```suggestion
	if t.Cfg.isModuleEnabled(IndexGateway) {
```

Reply:
I did not check the function implementation and thought it that it would also return true if the module is a dependency of the target. I think the function name is slightly misleading.

Of course, I will apply your suggestion.

Reply:
yeah, the naming is confusing. This one checks if the user explicitly enabled the module in the targets flag. There is also `isModuleActive` which checks if the module would be initialized.